src/core/timeout_middleware.py

```
@@ -52,17 +52,19 @@ async def __call__(
         timeout_seconds = self._get_tool_timeout(tool_name)
         
         start_time = time.time()
-        logger.debug(f"Starting tool '{tool_name}' (ID: {operation_id}) with {timeout_seconds}s timeout")
+        logger.info(f"🚀 TIMEOUT MIDDLEWARE: Starting tool '{tool_name}' (ID: {operation_id}) with {timeout_seconds}s timeout")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)